### ISSUE TITLE
to_yaml raises FileNotFoundError if file is just the filename

### ISSUE THREAD

**Issue by davesargrad** [fabiocaccamo/python-benedict#226]
The following code :
```
 if self.args.output:
            spec_app_dep.to_yaml(filepath=self.args.output)
```

works fine, if

- self.args.output is out/test1.yaml

it fails if:

- self.args.output is test.yaml

On failure it raises a FileNotFoundError exception:
`App Failure: (<class 'FileNotFoundError'>, FileNotFoundError(2, 'No such file or directory'), <traceback object at 0x7fb994c18e80>)`

In both cases the file is not there (and shouldnt be there) prior to the attempt to write it.

**Comment by fabiocaccamo** [comment]
@davesargrad nice catch, thank you for reporting it.

It's fixed in `0.28.2` version.

**Comment by davesargrad** [comment]
Youre so fast man. I love it.
